.travis.yml

```diff
@@ -17,20 +17,18 @@ cache:
   directories:
     - $HOME/.composer/cache
 
-before_script:
-  # Initialise Composer
-  - composer self-update -q
-  - composer global require hirak/prestissimo
```

Comment:
We use Composer cache: parallel downloads is god but helps less than it takes to download hirak/prestissimo